tensorrt_llm/_torch/modules/linear.py

```diff
@@ -181,6 +184,11 @@ def __init__(self,
         self.is_expert = is_expert
         self.use_custom_cublas_mm = use_custom_cublas_mm
 
+        self.linear_lora = LoraLayer(
```

Comment:
I have few concern about this part. Currently, the module name is unique. e.g., you will not see QKV on two parts in single layer. 

But we might have multiple linear layer and they will use same module names. It might lead to confusion when we try to select the lora weight. I remember we don't have such module now. Is there any reason to add it?

Reply:
I agree with 2.